### DISCUSSION TITLE
How does one subscribe to the blog?

### DISCUSSION THREAD

**Discussion by mfulton26** [vercel/turborepo#8488] [Help] [answered]
### Summary

I don't see an RSS or Atom feed available. Is there a way? I was hoping to use `/feed` in Slack.

**Comment by anthonyshew** [comment, marked as answer]
https://turbo.build/feed.xml

**Reply by mfulton26** [reply]
@anthonyshew it'd be great if the source page on turbo.build linked to the feed (e.g. via an alternate rel link). That's how I typically find Atom/RSS feeds (and programmatic tools often rely on this too). Thank you for the answer. I will use that. I tried `/feed.rss`, `/feed.atom`, `/rss`, `atom`, but not `feed.xml`. 🤦🏻

**Reply by anthonyshew** [reply]
Great idea. Made a PR to get that rolled out, thank you.
